.github/pull_request_template.md

```diff
@@ -2,56 +2,23 @@
 
 # Changes
 
-<!--
-Describe your changes here- ideally you can get that description straight from your descriptive commit message(s)!
-
-In addition, categorize the changes you're making using the "/kind" Prow command, example:
-
-/kind <kind>
-
-Supported kinds are: bug, cleanup, design, documentation, feature, flake, misc, question, tep
--->
+<!-- Describe your changes here- ideally you can get that description straight from your descriptive commit message(s)! -->
 
 # Submitter Checklist
 
 As the author of this PR, please check off the items in this checklist:
 
-- [ ] [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
-- [ ] [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
+- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
+- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
 - [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
 - [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
   functionality, content, code)
-- [ ] Release notes block below has been filled in
-(if there are no user facing changes, use release note "NONE")
+- [ ] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
+- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
+- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release
 
 # Release Notes
 
-<!--
-Describe any user facing changes here, or delete this block.
-
-Examples of user facing changes:
-- API changes
-- Bug fixes
-- Any changes in behavior
-- Changes requiring upgrade notices or deprecation warnings
-
-For pull requests with a release note:
-
-``` release-note
-Your release note here
-```
-
-For pull requests that require additional action from users switching to the new release, include the string "action required" (case insensitive) in the release note:
-
-``` release-note
-action required: your release note here
-```
-
-For pull requests that don't need to be mentioned at release time, use the `/release-note-none` Prow command to add the `release-note-none` label to the PR. You can also write the string "NONE" as a release note in your PR description:
-
 ``` release-note
```

Comment:
I would remove the space between the ticks and `release-note` - it's very hard to spot so many PRs fail the release notes check because of that